Sources/StreamVideo/Call.swift

```diff
@@ -238,7 +240,8 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
         notify: Bool = false,
         maxDuration: Int? = nil,
         maxParticipants: Int? = nil,
-        backstage: BackstageSettingsRequest? = nil
+        backstage: BackstageSettingsRequest? = nil,
+        hasVideo: Bool? = nil
```

Comment:
I think we should use just `video`, to be inline with the API and the other flags (although I agree `hasVideo` is more descriptive)